test_runner: exclude test files from code coverage by default (#55633)
nodejs/node: doc/api/cli.md

If both `--test-coverage-exclude` and `--test-coverage-include` are provided,
files must meet **both** criteria to be included in the coverage report.

> By default, the files being tested are excluded from code coverage. They can be explicitly
> included via this flag.
> 

Reviewer: ```suggestion
By default, the test files are excluded from code coverage. They can be explicitly
included via this flag.
```

> Test files are excluded by default. To include test files in code coverage,
> they must be explicitly included by using this parameter.

Reviewer 2: ```suggestion
By default, the files being tested are excluded from code coverage. They can be explicitly
included via this flag.
```
